app/javascript/components/RetrospectiveCreationForm.js

```diff
@@ -55,6 +57,8 @@ const RetrospectiveCreationForm = () => {
     setNewGroupName(name)
   }
 
+  const displayDescription = () => { return RETROSPECTIVE_DESCRIPTIONS[retrospectiveKind] }
```

Comment:
For functions composed of a single instruction, you don't need curly brackets and can use an implicit return:
 
```suggestion
  const displayDescription = () => RETROSPECTIVE_DESCRIPTIONS[retrospectiveKind]
```

Reply:
Thanks! Still not very familiar with JS syntax